core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -660,4 +690,44 @@ public RestConfig(ConfigDef definition) {
   public Time getTime() {
     return defaultTime;
   }
+
+  public static void validateHttpResponseHeaderConfig(String config) {
+    try {
+      // validate format
+      String[] configTokens = config.trim().split("\\s+", 2);
+      if (configTokens.length != 2) {
+        throw new ConfigException(String.format("Invalid format of header config for %s "
+                + "The format must be in [ation] [header name]:[header value]", config));
```

Comment:
Typo: `[action]`. I also think the overall message is a little redundant, though.
`"Invalid format of header config \"%s\". Expected` \"[action] [header name]:[header value]\""`